portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,194 @@
+                      faces by Deepfake techniques.
+                      Used these well-trained feature extractors and advanced pooling methods to build video
+                      classification models, which could identify fake videos.
+                      <br />
+                    </p>
+
+                    <v-col></v-col>
+
+                    <h3 class="text--primary"> Learn to Write Like You with a Few Examples </h3>
+                    <br />
+                    <p class="text--primary">
+                      Student project, Google Machine Learning Winter Camp 2020.
+                      <br />
+                      The main goal of this project is to learn to write Chinese characters like you with only a few
+                      examples,
+                      which means users can create their own fonts with a small amount of handwriting Chinese
+                      characters. 
+                      This project presents a novel GAN framework, based on pixel2pixel algorithm and
+                      utilizing the transfer learning idea to generate new style without retraining.
+                      <br />
+                      <a href="https://github.com/tonysy/FewShot_ChineseCharacter_GAN"> Code is Here. </a>
+                      <v-img max-width="600" src="images/mlcamp.png">
+                      </v-img>
+                    </p>
+
+                  </v-card-text>
+                </v-card>
+              </v-tab-item>
+              <v-tab-item>
+                <v-row>
+                  <v-col>
+                    <v-card>
+                      <v-container fluid>
+                        <v-row>
+                          <v-col v-for="n in 9" :key="n" class="d-flex child-flex" cols="4">
+                            <v-card flat tile class="d-flex">
+                              <v-img :src="`images/img${n}.jpg`" aspect-ratio="1" class="grey lighten-2">
+                                <template v-slot:placeholder>
+                                  <v-row class="fill-height ma-0" align="center" justify="center">
+                                    <v-progress-circular indeterminate color="grey lighten-5"></v-progress-circular>
+                                  </v-row>
+                                </template>
+                              </v-img>
+                            </v-card>
+                          </v-col>
+                        </v-row>
+                      </v-container>
+                    </v-card>
+                  </v-col>
+                </v-row>
+              </v-tab-item>
+            </v-tabs>
+          </v-card>
+          <v-col></v-col>
+          <v-card outlined>
+            <v-col></v-col>
+            <v-form method="post" action="/data">
+              <v-textarea name="comment" label="Add a comment..." outlined color="blue" height="50"
+                v-model="comment_content"></v-textarea>
```

Comment:
format:
```html
<v-textarea name="comment" label="Add a comment..." outlined color="blue" height="50" 
            v-model="comment_content">
</v-textarea>
```

Reply:
Seems the text is still no aligned?

Reply:
Sorry for my carelessness. Have aligned the text.